src/webui/modules/detail/index.jsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {Loading} from 'element-react';
 import isEmpty from 'lodash/isEmpty';
+import Prism from 'prismjs';
+import 'prismjs/themes/prism.css'
```

Comment:
How would we handle whether any user want's a different theme?

Reply:
Hi everyone. Thanks for the really fast response -- sorry I didn't get back to you all sooner but I was at the zoo all day yesterday with the kids...

* @juanpicado I deliberately left the theme unconfigurable while #357 is open. We _could_ serve all of the different themes statically and then import one of them into `index.html` depending on a configuration value. The downside of this is that we increase page weight of all pages, not just the detail pages, and we increase complexity. I would have thought it would be better to try to integrate theming of the syntax highlighting with whatever method we use for #357 when that's finalised.
* I've run a before-and-after bundle size check and the results are:
  * Bundle 3 `.js`: before 16kB, after 28kB.
  * Bundle 3 `.css`: before 8.8kB, after 11kB.
* We could consider vendoring out the prism library so we don't have to reload Prism every time we change our custom javascript.
* @sergiohgz Thanks for the testing tip. I've tried writing a snapshot-based unit test for this feature but I'm running into difficulties because the highlighting happens in `componentDidMount` after the readme contents have been retrieved. This doesn't get fired in the unit test and so I can't check that the code has been highlighted. Any tips would be appreciated!